runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/util/CloudObjects.java

```diff
@@ -50,19 +50,19 @@ private CloudObjects() {}
     return builder.build();
   }
 
-  public static CloudObject asCloudObject(Coder<?> coder) {
+  public static CloudObject standardCoderAsCloudObject(Coder<?> coder) {
     if (CODER_TRANSLATORS.containsKey(coder.getClass())) {
       // TODO: Make this cast less dangerous
-      return asCloudObject((StandardCoder<?>) coder);
+      return standardCoderAsCloudObject((StandardCoder<?>) coder);
```

Comment:
Add a test for the custom coder case showing that you can pass one in and then convert it back from the cloud object.